db/migrate/20211109141613_add_rdvs_users_notifications_flags.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+class AddRdvsUsersNotificationsFlags < ActiveRecord::Migration[6.0]
```

Comment:
Je constate que tu as opté pour l'ajout sur la table existante. Je continue de craindre la complexité de code et la forêt de `if` pour faire la distinction entre les `users` bénéficiant du RDV et les `users` à notifier qui ne sont pas les mêmes...

Et peut-être que tu as un plan en tête et que ça sera très clair, compréhensible et lisible :shrug: